tests/integration/test_hpc_libs.yaml

```diff
@@ -20,23 +20,25 @@ acts:
     name: "Test the is_container library"
     run-on: noble
     input:
-      - host-path: lib
-        path: lib
-      - host-path: dev-requirements.txt
-        path: dev-requirements.txt
+      - host-path: src
+        path: src
+      - host-path: pyproject.toml
+        path: pyproject.toml
+      - host-path: uv.lock
+        path: uv.lock
+      - host-path: README.md
+        path: README.md
```

Comment:
Is _README.md_ necessary here for the test to run correctly?

Reply:
Yes, because if you specify the `readme` field on the `pyproject.toml`, it is required to exist. I could also touch an empty file I guess.